Python/azureml-pipeline-databricks/README.md

```diff
@@ -0,0 +1,164 @@
+validation and scoring image deployment.
+
+In this sample, Databricks is used for feature engineering previously to building an ML model. A Databricks step
+can also be used for model training and model batch scoring.
+
+## Contents
+
+| File/folder             | Description                                                 |
+|-------------------------|-------------------------------------------------------------|
+| `README.md`             | This README file.                                           |
+| `azure-pipelines.yml`   | The Azure ML build and integration pipeline.                |
+| `ci_dependencies.yml`   | The image definition for the CI environment.                |
+| `code`                  | The Databricks feature engineering notebook and unit tests. |
+| `docs`                  | Images for this README file.                                |
+| `environment_setup`     | Pipelines and configuration for building the environment.   |
+| `ml_service`            | Python script for provisioning the Azure environment.       |
+| `tox.ini`               | Linting and unit test configuration.                        |
+
+## Running the sample
+
+### Getting the code
+
+Fork this repository within GitHub, or clone it into your Azure DevOps project.
+
+### Create an Azure DevOps organization
+
+We use Azure DevOps for running our MLOps pipeline with build (CI), ML training and scoring service release
+(CD) stages. If you don't already have an Azure DevOps organization, create one by
+following the instructions [here](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops).
+
+If you already have an Azure DevOps organization, create a [new project](https://docs.microsoft.com/en-us/azure/devops/organizations/projects/create-project?view=azure-devops).
+
+### Create an ARM service connection for Terraform
+
+The `DataOpsML ARM Connection` service connection is used by the [Azure DevOps pipeline](environment_setup/terraform-init-template.yml) to creates the Azure ML workspace and associated resources through Terraform. The pipeline requires an **Azure Resource Manager**
+[service connection](https://docs.microsoft.com/en-us/azure/devops/pipelines/library/service-endpoints?view=azure-devops&tabs=yaml#create-a-service-connection).
+
+Leave the **``Resource Group``** field empty.
+
+**Note:** Creating the ARM service connection scope requires 'Owner' or 'User Access Administrator' permissions on the subscription.
+You must also have sufficient permissions to register an application with
+your Azure AD tenant, or receive the ID and secret of a service principal
+from your Azure AD Administrator. That principal must have 'Contributor'
+permissions on the subscription.
+
+### Create a storage account for the Terraform state
+
+Create an Azure storage account. In the storage account, create a storage container named `terraformstate`.
```

Comment:
```suggestion
Create an Azure storage account. using the Portal interface or the CLI command below In the storage account, create a storage container named `terraformstate`.
```
```
location=[LOCATION]
rg=[RG_NAME]
storage_name=[UNIQUE_STORAGE_NAME]
az group create --location $location --name $rg
az storage account create --name $storage_name --resource-group $rg --location $location
az storage container create -name terraformstate --account-name $storage_name
```

Reply:
For standardization, linking to doc page rather than adding doc